src/battle.ts

```diff
@@ -2778,6 +2807,12 @@ class Battle {
 			this.log(args, kwArgs);
 			break;
 		}
+		case '-sideswitch': {
```

Comment:
```suggestion
		case '-swapsideconditions': {
```

On reflection, I think `sideswitch` is at least a little ambiguous.

Reply:
Sadly this doesn't match `sidestart` or `sideend` very well, but I think clarity is probably more important?